letsencrypt/DOCS.md

```diff
@@ -95,6 +114,41 @@ inwx_username: ''
 inwx_password: ''
 inwx_shared_secret: ''
 ```
+</details>
+
+### Configure certificate files
+
+The certificate files will be available within the "ssl" share after successful request of the certificates.
+
+By default other addons are referring to the correct path of the certificates.
+You can in addition find the files via the "samba" addon within the "ssl" share.
+
+For example, to use the certificates provided by this add-on to enable TLS on Home Assistant in the default paths, add the following lines to Home Assistant's main configuration file, `configuration.yaml`:
+
+```yaml
+# TLS with letsencrypt add-on
+http:
+  server_port: 443
+  ssl_certificate: /ssl/fullchain.pem
+  ssl_key: /ssl/privkey.pem
+```
+
+### Create & renew certificates
+
+The letsencrypt add-on crates the certificates once it is started: navigate to **Settings** -> **Add-ons**, pick the **Let's Encrypt** add-on, click the **START** button on the bottom. The add-on stops once the certificates are created.
+
+Certificates are not renewed automatically by the plugin. The add-on has to be started again to renew certificates: If the add-on is started again, it checks if the certificates are due for renewal. This us usually the case 30 days before the certificates due date. If the certificates are not due for renewal, the add-on terminates without changes. If the certificates are due for renewal, new certificates will be created.
+
+There are multiple ways how the add-on can be started to check/renew the certificates. One way to automate the certificate renewal it to use an automation, starting the add-on for example every night:
+
+1. Navigate in your Home Assistant frontend to **Settings** -> **Automations & Scenes**
+2. Click the button **CREATE AUTOMATION**
+3. Pick **Start with empty automation**
+4. Click **ADD TRIGGER**, pick **Time** and enter any time at the day
+5. Click **ADD ACTION**, pick **Call Service**, pick **Home Assistant Supervisor: Restart add-on.**, pick the add-on **Let's Encrypt**
+6. Click **SAVE**
```

Comment:
Instead of this block, I will prefer to be a bit more vague - just add a link to [HA docs](https://www.home-assistant.io/docs/automation/editor/) and instruct people to use the "Restart add-on" action on specific time. Otherwise it adds some documentation maintenance burden to these docs.

Reply:
I updated the corresponding part, please take a look.